src/app/ubs-admin/components/ubs-admin-cancel-modal/ubs-admin-cancel-modal.component.scss

```diff
@@ -0,0 +1,31 @@
+.container {
+  margin: 0;
+  padding: 0;
+  font-family: var(--primary-font);
+  width: 422px;
+}
+
+.question {
+  font-size: 16px;
+  color: var(--ubs-primary-grey);
+  padding: 1px 30px 20px 10px;
+}
+
+.disagree {
+  font-weight: bold;
+  font-size: 14px;
+  border: 1px solid var(--primary-green);
+  background: #fff;
+  color: #13aa57;
+  padding: 8px 28px;
+  margin-right: 25px;
+}
+
+.agree {
+  font-weight: bold;
+  font-size: 14px;
+  border: 1px solid var(--primary-green);
```

Comment:
You can unite this line like 
.disagree,
.agree {
    font-weight: bold;
    font-size: 14px;
    border: 1px solid var(--primary-green);
}
But it's optional, as you want.

Reply:
agree

Reply:
done